README.md

```diff
@@ -1,77 +1,28 @@
-
-### Spares
-
-The player has a spare if the knocks down all 10 pins with the two rolls of a frame. The bonus for that frame is the number of pins knocked down by the next roll (first roll of next frame).
-
-### 10th frame
-
-If the player rolls a strike or spare in the 10th frame they can roll the additional balls for the bonus. But they can never roll more than 3 balls in the 10th frame. The additional rolls only count for the bonus not for the regular frame count.
-
-    10, 10, 10 in the 10th frame gives 30 points (10 points for the regular first strike and 20 points for the bonus).
-    1, 9, 10 in the 10th frame gives 20 points (10 points for the regular spare and 10 points for the bonus).
-
-### Gutter Game
-
-A Gutter Game is when the player never hits a pin (20 zero scores).
+## My application
 
-### Perfect Game
+Currently, you can choose your score for each bowl, and the total for that frame will be calculated at the end of that frame (not including strike and spare scores.)
 
-A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
+I ran out of time to calculate a total and integrate spare and strike logic.
 
-In the image below you can find some score examples.
+## Personal Reflection
 
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
+This was a tough week for me. I started by thinking about what the application would look like, and had a complete mental block on how a person would interact with it. I was unable for some reason to see past just randomising the output of rolls, and only realised late Sunday evening of the first week that my interface could be interactive.
 
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+I struggle greatly with planning, and feel lost until I start coding, and I am not really sure how to approach full stack applications like this in terms of planning. Should I have planned out the interface first and worked back to the logic? I started on the logic first as this was what was most comfortable for me as I was not used to browser-based interaction.
 
-## Code Review
+I felt that my logic was quite clean, however the interface logic and jQuery is very messy, with an incredibly long function doing all the work.
 
-In code review we'll be hoping to see:
+I also could not figure out how to score strikes and spares properly, and ran out of time to do so.
 
-* All tests passing
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+## Key questions
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Note that referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want.
+How do I go about planning building this type of application? How do I break it down? How do I visualise it?
```

Comment:
Try to think about the simplest possible case for finishing a game of bowling - perhaps a game in which no pins are knocked down.  Think about how you could create a system which could handle this case.  Once it works, think about another simple case (a game of all ones?) and build upon your existing code.  If you keep repeating this until your code handles all the cases you can think of, you can gradually build up complexity, rather than trying to build a fully-realised, complex system in a single shot.